content/en/monitors/create/types/ci.md

```diff
@@ -158,6 +155,8 @@ This notification will only be sent for monitor alerts!
 {{/is_alert}}
 ```
 
+**Note**: A monitor that uses a formula for its evaluation query will keep the last state (e.g. Alert) if there is no data by default.
```

Comment:
That's not really true, is it? I definitely applies if I do `a / b * 100` but what if I do `a + b`? Not saying that it makes sense creating a monitor with `a + b` but I think it's good to be specific here.

Reply:
Based on this change https://github.com/DataDog/dd-go/pull/76284, all F&F monitors will keep the last state if there is no data. 

I've tested `a+b` and it works like this https://admin.us1.prod.dog/monitors/cluster/realtime/org/2/monitor/101181165?from_ts=1667917463516&to_ts=1667918363516

Reply:
It's not on "no data", but when the formula cannot be evaluated (i.e. `a/b` and `b=0`). If there's no data and the formula evaluates to `0`, the query will not keep the last state but use `0` for evaluation.

I think we should rewrite this whole section. We should state that if the query is a count and there is no data, it will evaluate to 0, potentially resolving immediately. Because of this, Datadog recommends using a formula like `(number of failures)/(number of all events)`, because if `(number of all events) = 0`, the division `x/0` cannot be evaluated and the monitor will keep the last known state.